### ISSUE TITLE
MD036 gets confused by parenthesis

### ISSUE THREAD

**Issue by VideoPlayerCode** [DavidAnson/markdownlint#258]
Try this document:

```markdown
# Example

## Example 1

Paragraph 1.

Paragraph 2.

Paragraph 3.

_(Note: Here is a side note which will get flagged by rule MD036.)_

## Example 2

Nothing...

```

I decided to avoid using parenthesis, since that fixes the linting problem. But it's still weird that it gives such an incorrect linting result in the parenthesis case. :-)

**Comment by DavidAnson** [comment]
I don’t think there is an issue here. The relevant part of the rule description:

> Note: This rule looks for single line paragraphs that consist entirely of emphasized text. It won't fire on emphasis used within regular text, multi-line emphasized paragraphs, or paragraphs ending in punctuation (normal or full-width).

MD036 fires in your example above because the conditions above are met. MD036 does not fire when the parenthesis are removed because then the paragraph ends in a punctuation character (‘.’).

More: https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md036---emphasis-used-instead-of-a-heading
